src/renderer/webtorrent.js

```diff
@@ -54,11 +54,16 @@ const VERSION_STR = VERSION.match(/([0-9]+)/g)
  */
 const VERSION_PREFIX = '-WD' + VERSION_STR + '-'
 
+/**
+ * Generate an ephemeral peer ID each time.
+ * TODO: once there are around 2^24 = ~8 million WebTorrent Desktops online at the same time,
+ *       ID collisions will start happening. Birthday paradox. Can we use more than six bytes?
```

Comment:
The peer id must be 20 bytes. By convention, 8 of those bytes are used to identify the client and version (for example, `-WD0100-` for WebTorrent Desktop v1.0). That leaves 12 bytes for the rest of the id.

There's no requirement that the ID use printable characters, though that's really nice to have for debugging. That's why we're doing 6 bytes, but printing them as hex (which uses 12 bytes).

We could add entropy and still keep this printable by changing it to base64:

```js
crypto.randomBytes(9).toString('base64')
```

Thoughts?